packages/angular/projects/clr-angular/src/forms/common/abstract-container.ts

```diff
@@ -15,6 +15,10 @@ import { ControlClassService } from './providers/control-class.service';
 import { Subscription } from 'rxjs';
 import { IfControlStateService, CONTROL_STATE } from './if-control-state/if-control-state.service';
 
+export interface ClassOptions {
+  supportSuccess?: boolean;
```

Comment:
This is unnecessary, if the abstract has a reference to the success component you can simply check for it and get rid of this